packages/http-server/src/__tests__/body-params-validation.spec.ts

```diff
@@ -796,5 +860,20 @@ describe('body params validation', () => {
         expect(response.status).toBe(200);
       });
     });
+
+    describe('valid multipart form data parameter provided', () => {
+      test('returns 422', async () => {
+        const response = await makeRequest('/multipart-form-data-body-required', {
+          method: 'POST',
+          body: new URLSearchParams({
+            status: 'new',
+          }).toString(),
+          headers: { 'content-type': 'multipart/form-data' },
+        });
+
+        expect(response.status).toBe(200);
```

Comment:
the test says it returns 422, but the assertion says it returns 200. What was the intent for this test? It seems like you're trying to test when passing an invalid multipart form data since the body is using URLSearchParams which is not multipart form data.